### PR TITLE
Add a --label flag to mc admin bucket remote add command

### PR DESCRIPTION
**Pull request by poornas**
This label will be used in lieu of transition storage class for [ilm transition](https://github.com/minio/minio/pull/10565)

### CODE REVIEW

**Review by kannappanr — APPROVED** [review]
LGTM

File: cmd/admin-bucket-remote-add.go [hunk 1]
```diff
@@ -82,7 +86,7 @@ EXAMPLES:
      {{.DisableHistory}}
      {{.Prompt}} {{.HelpName}} myminio/srcbucket \
                  https://foobar:foo12345@minio2:9000/replicabucket \
-                 --service "replication" --region "us-west-1"
+                 --service "replication" --region "us-west-1" --label "replica1"
```
Comment (harshavardhana) [inline cmd/admin-bucket-remote-add.go:89]:
```suggestion
                 --service "replication" --region "us-west-1" --label "hdd-tier"
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]
I am not sure why is this service called `replication` shouldn't the examples be called `service=transition` ?

**Comment by poornas** [comment]
> I am not sure why is this service called `replication` shouldn't the examples be called `service=transition` ?

The ilm transition PR needs to be in before I can do that - currently only valid service is replication on minio master branch

**Comment by harshavardhana** [comment]
> The ilm transition PR needs to be in before I can do that - currently only valid service is replication on minio master branch

Is label feature also available for replication ? if so how is it used?

**Comment by poornas** [comment]
> > The ilm transition PR needs to be in before I can do that - currently only valid service is replication on minio master branch
> 
> Is label feature also available for replication ? if so how is it used?

no, this is just a tag to identify the remote and is ignored by replication. It is repurposed in ilm transition to also sub as the storage-class name.

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**